tests/participant/test_resources.py

```diff
@@ -106,6 +112,7 @@ def test_put_participant(self):
         participant = resp['results']
         self.assertEqual(p.kf_id, participant['kf_id'])
```

Comment:
This is probably my fault from before, but these asserts need to change to:
```python
self.assertEqual(p.kf_id, participant['kf_id'])
self.assertEqual(p.external_id, body['external_id'])		          
self.assertEqual(p.consent_type, body['consent_type'])
```
The reason why is bc this test should not actually be passing, since the participant resource put method hasn't been updated with the new fields. Its only passing because the participant.<new field> and p.<new field> are probably both None.

So change these assert statements and then also make sure to update the participant resource put method. We should make that deserialization dynamic in the near future so that we don't have to manually deserialize fields into the put methods of resources.